modules/claims_api/spec/workers/ews_vbms_sidekiq_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'claims_api/ews_vbms_sidekiq'
+
+RSpec.describe ClaimsApi::EwsVBMSSidekiq do
+  let(:dummy_class) { Class.new { extend ClaimsApi::EwsVBMSSidekiq } }
```

Comment:
just curious why the `dummy_class` instead of using `ClaimsApi::EwsVBMSSidekiq` directly?

Reply:
i was replicating the VBMS_sidekiq_spec for consistency